msg/scratch_msgs.js

```diff
@@ -1,13 +1,31 @@
-// This file was automatically generated.  Do not modify.
+import * as Blockly from 'blockly';
 
-'use strict';
-
-goog.provide('Blockly.ScratchMsgs.allLocales');
-
-goog.require('Blockly.ScratchMsgs');
+const ScratchMsgs = {
+  currentLocale_: 'en',
+  setLocale: function(locale) {
+    if (Object.keys(this.locales).includes(locale)) {
+      this.currentLocale_ = locale;
+      Blockly.Msg = Object.assign({}, Blockly.Msg, this.locales[locale]);
+    } else {
+      // keep current locale
+      console.warn('Ignoring unrecognized locale: ' + locale);
+    }
+  },
+  translate: function(msgId, defaultMsg, useLocale) {
```

Comment:
Curious, did this method used to live somewhere else?

Reply:
Yes, they're verbatim from core/scratch_msgs.js.